src/steps/lead-field-equals.ts

```diff
@@ -0,0 +1,51 @@
+/*tslint:disable:no-else-after-return*/
+
+import { BaseStep, Field, StepInterface } from '../core/base-step';
+import { Step, FieldDefinition, StepDefinition, RunStepResponse } from '../proto/cog_pb';
+
+export class LeadFieldEquals extends BaseStep implements StepInterface {
+
+  protected stepName: string = 'Check a field on a Dynamics CRM Lead';
+  protected stepExpression: string = 'the (?<field>[a-zA-Z0-9_]+) field on dynamics lead (?<email>.+) should be (?<expectedValue>.+)';
```

Comment:
- [ ] Consistency nit: `the (?<field>[a-zA-Z0-9_]+) field on dynamics crm lead (?<email>.+) should be (?<expectedValue>.+)`